mage_ai/io/oracledb.py

```diff
@@ -10,11 +10,19 @@
 
 
 class OracleDB(BaseSQL):
-    def __init__(self, user, password, host, port, verbose: bool = False, **kwargs) -> None:
+    def __init__(self,
+                 user,
+                 password,
+                 host,
+                 port,
+                 service,
```

Comment:
Could you change service to service_name? That's how [python-oracledb](https://python-oracledb.readthedocs.io/en/latest/user_guide/connection_handling.html) expects this parameter name

![Screenshot from 2023-07-24 17-47-47](https://github.com/mage-ai/mage-ai/assets/14100959/130a3f7a-7955-4dad-93dc-5a52c9579696)